runtime/vm/JFRChunkWriter.cpp

```diff
@@ -208,15 +228,15 @@ VM_JFRChunkWriter::writeThreadStateCheckpointEvent()
 		_bufferWriter->writeLEB128(UTF8);
 
 		/* string length */
-		U_32 len = strlen(threadStateNames[i]);
+		UDATA len = strlen(threadStateNames[i]);
 		_bufferWriter->writeLEB128(len);
 
 		/* write string */ // need to think about UTF8 enconding
 		_bufferWriter->writeData((U_8 *)threadStateNames[i], len);
 	}
 
 	/* write size */
-	_bufferWriter->writeLEB128PaddedU32(dataStart, _bufferWriter->getCursor() - dataStart);
+	writeCheckpointSize(dataStart);
```

Comment:
This has been removed in favour of using `writeEventSize()`.

Reply:
Sorry, fixed.